includes/multi-currency/client/settings/multi-currency/enabled-currencies-list/index.js

```diff
@@ -13,55 +14,37 @@ import { createInterpolateElement } from '@wordpress/element';
  * Internal dependencies
  */
 import './style.scss';
-import {
-	useCurrencies,
-	useDefaultCurrency,
-	useEnabledCurrencies,
-} from 'multi-currency/data';
+import { useCurrencies, useEnabledCurrencies } from 'multi-currency/data';
 
 import EnabledCurrenciesList from './list';
 import EnabledCurrenciesListItem from './list-item';
 import EnabledCurrenciesListItemPlaceholder from './list-item-placeholder';
 import EnabledCurrenciesModal from './modal';
 import { SettingsSection } from 'multi-currency/interface/components';
 
-const EnabledCurrenciesSettingsDescription = () => {
-	const LEARN_MORE_URL =
-		'https://woocommerce.com/document/woopayments/currencies/multi-currency-setup/#enabled-currencies';
-
-	return (
-		<>
-			<h2>{ __( 'Enabled currencies', 'woocommerce-payments' ) }</h2>
-			<p>
-				{ createInterpolateElement(
-					sprintf(
-						__(
-							'Accept payments in multiple currencies. Prices are converted ' +
-								'based on exchange rates and rounding rules. <learnMoreLink>' +
-								'Learn more</learnMoreLink>',
-							'woocommerce-payments'
-						),
-						LEARN_MORE_URL
+const EnabledCurrenciesSettingsDescription = () => (
+	<>
+		<h2>{ __( 'Enabled currencies', 'woocommerce-payments' ) }</h2>
+		<p>
+			{ createInterpolateElement(
+				__(
+					'Accept payments in multiple currencies. Prices are converted ' +
+						'based on exchange rates and rounding rules. <learnMoreLink>' +
+						'Learn more</learnMoreLink>',
+					'woocommerce-payments'
+				),
+				{
+					learnMoreLink: (
+						<ExternalLink href="https://woocommerce.com/document/woopayments/currencies/multi-currency-setup/#enabled-currencies" />
 					),
-					{
-						learnMoreLink: (
-							// eslint-disable-next-line jsx-a11y/anchor-has-content
-							<a
```

Comment:
The previous link was just using a `a` component, but opening in a new page.
Ensuring that we use the `ExternalLink` component, which adds the "Opens in a new tab" text for a11y.